test/codegen-generated-classes-test/src/test/java/software/amazon/awssdk/services/protocolquery/MoveQueryParamsToBodyTest.java

```diff
@@ -88,14 +96,49 @@ public void customInterceptor_additionalQueryParamsAdded_paramsAlsoMovedToBody()
 
         String contentString = IoUtils.toUtf8String(requestContent.newStream());
 
-        assertThat(contentString).contains("CustomParamName=CustomParamValue");
+        assertThat(contentString).contains(CUSTOM_PARAM_NAME + "=" + CUSTOM_PARAM_VALUE);
+    }
+
+    @Test
+    public void requestOverrideConfiguration_additionalQueryParamsAdded_paramsAlsoMovedToBody() throws IOException {
```

Comment:
Can we add one e2e test where we don't add the params, and also cover the async client for both?

Reply:
added additional tests